src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTORY = 0.75F;
+    private Node<K, V>[] table;
+    private int size;
+    private float loadFactor;
+    private float threshold;
```

Comment:
thereshold it's variable wich says when we should resize table. declare it as int and cast float to int when calculating

Reply:
ok